x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/find_ml_signals.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import dateMath from '@elastic/datemath';
+
+import { AlertServices } from '../../../../../../../plugins/alerting/server';
+
+import { getAnomalies } from '../../machine_learning';
+
+export const findMlSignals = async (
+  jobId: string,
+  anomalyThreshold: number,
+  from: string,
+  to: string,
+  callCluster: AlertServices['callCluster']
+) => {
+  const params = {
+    jobIds: [jobId],
+    threshold: anomalyThreshold,
+    earliestMs: dateMath.parse(from)!.valueOf(),
+    latestMs: dateMath.parse(to)!.valueOf(),
```

Comment:
I'd rather have:

```ts
    earliestMs: dateMath.parse(from)?.valueOf() ?? 0,
    latestMs: dateMath.parse(to)?.valueOf() ?? 0,
```

Or use `-1` instead of 0 instead of turning off type assertion. This would be better than a potential crashing if we can't parse the `from` and `to` since we don't have very strict checks against it.